applications/simulate_prod.py

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/python3
+
+"""
+    Summary
+    -------
+    This application is used to run simulations for productions (typically on the grid).
+    It allows to run a Paranal (CTA-South) or La Palma (CTA-North) array layout simulation
```

Comment:
Further down, it is `CTAO-Site`, maybe we should consistently use `CTAO` (what a useless comment..)

Reply:
Yes, fixed.